libcudacxx/include/cuda/std/__floating_point/constants.h

```diff
@@ -39,13 +40,24 @@ template <__fp_format _Fmt>
 {
   static_assert(__fp_has_inf_v<_Fmt>, "The format does not support infinity");
 
-  return __fp_exp_mask_v<_Fmt>;
+  return static_cast<__fp_storage_t<_Fmt>>(__fp_exp_mask_v<_Fmt> | __fp_explicit_bit_mask_v<_Fmt>);
```

Comment:
Question: are we sure the compiler can elide this operation or should we use an `if constexpr` too circumvent it?